collectors/osv/tests/test_collectors.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from collectors.osv.collectors import OSVCollector
+from osidb.models import Snippet
+
+pytestmark = pytest.mark.integration
+
+
+class TestNVDCollector:
+    @pytest.mark.vcr
+    def test_collect_osv_record(self):
+        """Test fetching a single OSV record."""
+        osvc = OSVCollector()
+        osvc.snippet_creation_enabled = True
+        osvc.collect(osv_id="GO-2023-2400")
+
+        snippet = Snippet.objects.last()
+        assert snippet.external_id == "GO-2023-2400/CVE-2023-50424"
+        assert snippet.content["references"]
+        assert snippet.content["cve_id"] == "CVE-2023-50424"
+
+    @pytest.mark.vcr
+    def test_collect_multi_cve_osv_record(self):
+        """Test fetching a single OSV record that points to multiple CVEs."""
+        osvc = OSVCollector()
+        osvc.snippet_creation_enabled = True
+        osvc.collect(osv_id="PYSEC-2022-245")
+
+        snippet_1 = Snippet.objects.get(external_id="PYSEC-2022-245/CVE-2022-36359")
```

Comment:
Could you please add `assert Snippet.objects.count() == 2`?

Reply:
Sure!